### ISSUE TITLE
Switch from uuid4 to uuid7 (newer version). Research what the added benefits and consequences are.

### ISSUE THREAD

**Issue by TNO-SlaFleur** [Project-OMOTES/omotes-sdk-protocol#5]

**Comment by WouterSpaak** [comment]
I don't think this would need any work in the context of this repository. uuidv4 and uuidv7 are identical in the sense that they follow the exact same shape. The main benefit of uuidv7 is that they're time-sortable: the first few bytes are a timestamp.

As for protobuf models, that doesn't really matter as there isn't a special `uuid` type. We've been pushing `Job`s to OMOTES with uuidv7 as id without problems. I don't really think there's any part of the system that'll choke on uuidv7, as they're interchangeable with uuidv4. It's mostly beneficial in databases as PK, because records inserted at the same time are close together in index pages.
